composer.json

```diff
@@ -12,7 +12,13 @@
 	"require": {
 		"php": "~7.1",
 		"phpstan/phpstan": "^0.11",
-		"nikic/php-parser": "^4.0"
+		"nikic/php-parser": "^4.0",
+		"doctrine/orm": "^2.5",
+		"doctrine/common": "^2.7",
+		"doctrine/annotations": "^1.5",
+		"doctrine/persistence": "^1.1",
+		"doctrine/dbal": "^2.5",
+		"doctrine/event-manager": "^1.0"
```

Comment:
I'm quite unconvinced of the need for this - composer-require-checker demands this, but actually if you dont configure 'mapping' you wont need any of this, and if you do, you should have it installed anyway

Reply:
I don't understand why any of the changes in composer.json were needed.

Reply:
Its because of the code that loads/creates the entity manager, but if we use the approach in #40 this wont be needed.